integration_test/install_test.go

```diff
@@ -132,7 +148,23 @@ func TestKrewInstall_OnlyArchive(t *testing.T) {
 	}
 }
 
-func TestKrewInstall_PositionalArgumentsAndManifest(t *testing.T) {
+func TestKrewInstall_ManifestArgsAreMutuallyExclusive(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+	srv, close := localTestServer()
+	defer close()
+
+	if err := test.Krew("install", validPlugin,
```

Comment:
This command does not fail due to incompatible `--manifest` and `--manifest-url`, but because it mixes positional args and `--manifest*`. This is also covered by the next test-case, and I think it will not pass, if `validPlugin` is removed.

Reply:
Ping @ahmetb. If I change the test to this:
```go
if err := test.Krew("install",
		"--manifest", filepath.Join("testdata", fooPlugin+constants.ManifestExtension),
		"--manifest-url", srv+"/"+validPlugin+constants.ManifestExtension,
		"--archive", filepath.Join("testdata", fooPlugin+".tar.gz")).
		Run(); err == nil {
		t.Fatal("expected mutually exclusive arguments to cause failure")
}
```
It does not fail, but it should.

Iow, the `--manifest` and `--manifest-url` options are not mutually exclusive, in contrast to what the test case claims.

Reply:
thanks, should be fixed now.